### DISCUSSION TITLE
Adding Submoduls in NetBox

### DISCUSSION THREAD

**Discussion by yousefii** [netbox-community/netbox#12326] [Help Wanted!]
Hello;
I just started using NetBox and I have an issue with adding modules.
Actually, I have a Module that has 2 submodules. When I add this module to a device I can't add a daughterboard to the main modules.
 How can I add this Module and then add 1 daughterboard to it?
 Is that impossible or have I missed something here?

**Comment by kkthxbye-code** [comment]
That is not currently possible, there's a feature request for it.

**Reply by yousefii** [reply]
Thank you for your support.
Could you tell me please, how can I request for that?

**Comment by candlerb** [comment]
For now, you'll need to either use inventory items (which can be associated with components, e.g. interfaces), or 'flatten' the top-level module. See #12211 which was basically the same question.
